starter_code/src/movies.js

```diff
@@ -1,20 +1,67 @@
 /* eslint no-restricted-globals: 'off' */
-// Turn duration of the movies from hours to minutes 
+// Turn duration of the movies from hours to minute
 
 
 // Get the average of all rates with 2 decimals 
+function ratesAverage(movies)
+{var sumRates = movies.reduce(function(sum, movie){
+  var temp = parseFloat(movie.rate);
+  return sum + temp;
+}, 0);
+return Math.floor((sumRates/movies.length)*100)/100;
+}
 
+ratesAverage(movies);
 
 // Get the average of Drama Movies
+var drama = [];
+movies.filter(function(movie){
+  if (movie.genre == "Drama"){
+      drama.push(movie);
+  }
+})
+function dramaMoviesRate(drama){
+ var dramaAverage = ratesAverage(drama);
+ console.log(" the average drama movies rate is " + dramaAverage );
+}
```

Comment:
Good job on reusing the ratesAverage function but what what would happen if I called dramaMoviesRate with a different array than the one from movies? Make sure that you are always working with the data you receive.